pom.xml

```diff
@@ -55,13 +55,13 @@
     <dependency>
       <groupId>com.github.jnr</groupId>
       <artifactId>jffi</artifactId>
-      <version>1.2.11</version>
+      <version>1.2.13</version>
```

Comment:
I will modify this to use the 1.2.13 snapshot until jffi has another release (this weekend or early next week).